src/Velociraptor.js

```diff
@@ -1,6 +1,5 @@
 import React from "react";
 import ReactDom from "react-dom";
-import Velocity from "velocity-animate";
```

Comment:
Perhaps this `Velociraptor` class should be renamed now that it no longer depends on Velocity...? Just a passing thought, not quite sure what a "good" name would be...

Reply:
I have renamed it to `NodeAnimator`. Not convinced this is the best name ever. But it will at least rightfully make it agnostic of the library (or lack of) it uses